findimports.py

```diff
@@ -30,6 +30,8 @@
                         --duplicate)
   -N, --noext           omit external dependencies
   -p, --packages        convert the module graph to a package graph
+  -pE, --package-externals
+                        convert external modules to a packages.
```

Comment:
Whoa, I didn't know that argparse supported multi-single-letter argument names like this.

(Does it really?)

Reply:
apparently so.